core/optionsx.h

```diff
@@ -478,7 +478,7 @@
      * later.
      */
     /* FIXME i#1621: NYI on ARM, partly implemented on AArch64 */
```

Comment:
Should this be updated?

Reply:
I've updated the FIXMEs. I've also left the default for AArch64 at 1 for now, until the FIXMEs are resolved.